packages/core/src/extensions/DraggableBlocks/DraggableBlocksPlugin.tsx

```diff
@@ -110,6 +128,7 @@ function dragStart(e: DragEvent, view: EditorView) {
     e.dataTransfer.setData("text/plain", text);
     e.dataTransfer.effectAllowed = "move";
     const block = getDraggableBlockFromCoords(coords, view);
+    // TODO: Fix drag image
```

Comment:
is this easy to fix?

Reply:
As it turns out, it's quite an annoying one. The `setDragImage` function takes a single HTML element as an argument, and that element must be visible in the browser when the drag starts. With only 1 selected node that's no problem, but with multiple not so much since they're no longer a single element, and you can't use the parent element since not all children might be selected. So the only real solution is to create a div somewhere the user can't see with all the selected nodes in it, and use that for the image. It seems quite hacky to me but AFAIK there really isn't really another option.